docs/standard/library-guidance/sourcelink.md

```diff
@@ -29,9 +29,9 @@ You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuG
 > * <xref:System.Diagnostics.DebuggerStepThroughAttribute> instructs the debugger to step through the code instead of stepping into the code.
 > * <xref:System.Diagnostics.DebuggerBrowsableAttribute> controls whether a member is displayed in the debugger variable windows.
 
-**✔️ CONSIDER** including symbol files (`*.pdb`) in the NuGet package.
+**✔️ CONSIDER** publishing symbol files (`*.pdb`).
 
-> Ordinarily, you'd publish symbol files in a [symbol package](./nuget.md#symbol-packages). Currently the main public host for symbol packages doesn't support the portable symbol files (`*.pdb`) created by SDK-style projects, and symbol packages aren't useful.
+> Read more about [symbol files and symbol packages](./nuget.md#symbol-packages).
```

Comment:
Our boilerplate for this kind of link is something like For more information, see or For more information about x, see...

```suggestion
> For more information about symbol files and symbol packages, see [Symbol packages](./nuget.md#symbol-packages).
```